src/js/component.js

```diff
@@ -95,7 +95,9 @@ class Component {
     if (options.evented !== false) {
       // Make this an evented object and use `el_`, if available, as its event bus
       evented(this, {eventBusKey: this.el_ ? 'el_' : null});
+
       if (this.player_) {
+        this.handleLanguagechange = this.handleLanguagechange.bind(this);
```

Comment:
With this change, we create a new function signature for the component that overloads `handleLanguagechange` and we bind it to it; The problem was that every component shares the handler stored in the player and if one removes the handler it will be removed for every component.

Reply:
Classic 😆